R/cf.R

```diff
@@ -842,6 +842,12 @@ add.cf <- function(cf1, cf2, a = 1.0, b = 1.0) {
   add.cf(cf1, cf2, a = 1.0, b = -1.0)
 }
 
+#' @rdname slash-.cf
```

Comment:
docu is a bit sparse...! And why `slash-.cf`?

Reply:
The documentation is sparse because it gets documented together with `/.cf` this way. And the division function is well documented.